inventory/vagrant/group_vars/webserver/general.yml

```diff
@@ -1,3 +1,5 @@
 ---
 
 webserver_app: yes
+webserver_app_jwt_key_path: /home/ubuntu/auth
```

Comment:
I saw a few comments about allowing multi platform approaches for our ansible playbook. Where would this one be if using Centos/Redhat?

Reply:
I guess it would be where ever you wanted to store these keys. Could be `/etc/ssl` if you so choose, but you'd have to make that directory. This directory is generated when the SSL keys are made, so all I am doing is copying the variable up to a higher point to allow both roles to access it the same.

Reply:
Then i guess we need to make this platform independent, you run Redhat right? 
This here is just for ubuntu:
https://github.com/Islandora-Devops/claw-playbook/blob/3efb1326addd71f0933d7945ea1bddf15cd0e7cc/inventory/vagrant/group_vars/tomcat.yml#L29
Mostly because every sprint issue we have right now says we should
try supports both Ubuntu and CENTOS. Just a comment because seems like that is one of the advantages of ansible at all (other than learning a new pseudo language we will forget if not using it on a daily basis)

Reply:
@jonathangreen @whikloj probably 
```YML
webserver_app_jwt_key_path:  {{ fcrepo_syn_sites[path] | default(/home/ubuntu/auth) }}
```
Not sure about the fcrepo_syn_sites: path: nomenclature (way of accesing) but i would for sure think we should reuse as much vars as possible